### PR TITLE
Prepare RGBD dataset 

### PR DESCRIPTION
**Pull request by markus-hinsche**
Feature: https://dev.azure.com/cgmorg/ChildGrowthMonitor/_workitems/edit/1507

**Changes:**
- ensure that `labels.csv` has the same amount of pickle files (don't include filtered or invalid into labels.csv)
- Fix: Handle InvalidDevicePoseError for rgbd 
- Release cgm-ml-common
- clean timestamp from spaces, colons, and undesired dots
- Refactor rotate_and_load_depthmap_with_rgbd into separate function
- Move EDA notebook from anon-rgbd-5k to anon-rgbd-5k-scans

**Docs:**
- I documented the [Dataset creation steps](https://dev.azure.com/cgmorg/ChildGrowthMonitor/_wiki/wikis/ChildGrowthMonitor.wiki/210/Preprocessing-steps-while-dataset-creation) as a checklist for future dataset creations
- Add docs visualization for dataset creation detailed steps

**Resulting datasets**

RGBD trainset: `anon-rgbd-5k-100and102`
- Total no. of artifacts: 5398
- Total no. of unique scan ids: 573
- Total no. of unique person ids: 152
- Dataset Creation processing takes 5 minutes

RGBD testset: `anon-rgbd-1k-100and102-testset`
- Total no. of artifacts: 1452
- Total no. of unique scan ids: 156
- Total no. of unique person ids: 60
- 1.2min

### CODE REVIEW

File: cgmml/data_utils/dataset_EDA/anon-rgbd-5k-100and102/EDA_anon-rgbd-5k-100and102.ipynb [hunk 1]
```diff
@@ -0,0 +1,1165 @@
+{
```
Comment (jasminziegler) [inline cgmml/data_utils/dataset_EDA/anon-rgbd-5k-100and102/EDA_anon-rgbd-5k-100and102.ipynb:1]: <p>same as in the other notebook, not clear to me why this has moved into the notebook from the utils. If it has an advantage in here that I don't see, we can remove the "find_outliers" from the import statement in the top. We could then think about following a universal approach for this matter in all EDA notebooks if we have decided on one alternative.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Welthungerhilfe/cgm-ml/pull/478/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Comment (markus-hinsche) [reply cgmml/data_utils/dataset_EDA/anon-rgbd-5k-100and102/EDA_anon-rgbd-5k-100and102.ipynb:1]: <p>You are right, I will change this</p>

**Review by jasminziegler — APPROVED** [review]
lgtm! the dataset creation steps are very helpful for my understanding. In case it magically disappears again: I left a comment concerning find_outliers function in reviewnb, not sure why the function moved from utils file into the notebook. Do you see an advantage there?

**Review by shanky0009 — APPROVED** [review]
LGTM

**Pull request merged**